packages/gx-agent/src/agent/GXComplianceClient.ts

```diff
@@ -144,19 +147,20 @@ export class GXComplianceClient implements IAgentPlugin {
       },
       context
     )
-    console.log(selfDescription.hash)
     const uniqueVP = await this.credentialHandler.issueVerifiablePresentation(
       {
         challenge: GXComplianceClient.getDateChallenge(),
         keyRef: signInfo.keyRef,
         verifiableCredentials: [selfDescription.verifiableCredential],
         domain: signInfo.participantDomain,
+        persist: args.persist ? args.persist : false,
       },
       context
     )
     const verifiableCredentialResponse = (await this.acquireComplianceCredential(
       {
         verifiablePresentation: uniqueVP.verifiablePresentation,
+        show: args.show ? args.show : false,
```

Comment:
Not sure why this is needed everywhere, simply pass in args.show

Reply:
changed to just: `args.show`